Final Project App/Final Project App/ViewController.swift

```diff
@@ -30,24 +28,20 @@ class ViewController: UIViewController, UITableViewDataSource {
         NotificationCenter.default.addObserver(self, selector: #selector(ViewController.keyboardWillShow), name: NSNotification.Name.UIKeyboardWillShow, object: nil)
         NotificationCenter.default.addObserver(self, selector: #selector(ViewController.keyboardWillHide), name: NSNotification.Name.UIKeyboardWillHide, object: nil)
         
-        self.InputTextField.delegate = self //Assign Text Field Delegate
-        self.tableView.dataSource = self // Assign Table view data source
+        inputTextField.delegate = self //Assign Text Field Delegate
+        tableView.dataSource = self // Assign Table view data source
         getData() // Call get data function on initial load
-        self.hideKeyboard() // Hide keyboard when tapped
+        hideKeyboard() // Hide keyboard when tapped
         tableView.contentInset = UIEdgeInsetsMake(0.0, 0.0, 30.0, 0.0) // Inset table view scroll to move from behind the keyboard
-        repeatTimer =   Timer.scheduledTimer(timeInterval: 5, target: self,
-                                             selector: #selector(getData), userInfo: nil, repeats: true) // Set timer to get new messages every 5 seconds
-    }
-    
-    //    Mark: Animate viewWillAppear
-    override func viewWillAppear(_ animated: Bool) {
-        super.viewWillAppear(animated)
+        getFromServerTimer =   Timer.scheduledTimer(timeInterval: 5, target: self,
```

Comment:
Timer.scheduledTimer(withTimeInterval 5, repeats: true) { _ in self.getData() }

Not strictly necessary, but as a general preference closures > selectors.